tools/env/dev.ts

```diff
@@ -4,7 +4,7 @@ const DevConfig: EnvConfig = {
   ENV: 'DEV',
   RMMAPI: 'https://testdata.nist.gov/rmm/',
   SDPAPI: 'https://testdata.nist.gov/sdp/',
-  PDRAPI: 'https://testdata.nist.gov/pdr/',
+  PDRAPI: 'https://testdata.nist.gov/od/id/',
   DISTAPI: 'https://testdata.nist.gov/od/',
```

Comment:
DISTAPI url is being used in PDR, yes it should be /od/ds, I think she is appending ds in the code